src/main/resources/assets/moremusic/lang/pl_pl.json

```diff
@@ -1 +1,23 @@
-
+{
+  "text.autoconfig.moremusic.title": "More Music Config (Upewnij się, że uruchomiłeś ponownie komputer, aby zmiany zaczęły obowiązywać!)",
```

Comment:
![image](https://user-images.githubusercontent.com/35371030/171909180-12f3a4c6-de18-453c-977b-fbced710a7ab.png)
This should say 'restart your game' rather than 'restart your computer' - admittedly the English string is kind of unclear on this, I should probably change that.